### PR TITLE
Add Skin Command

### PR DESCRIPTION
**Pull request by ConorTheDev**
This command has one argument; the username of the skin you want to get
Screen Shot 2019-07-13 at 20 20 34

It's a pretty simple but nice command :)

### CODE REVIEW

File: cogs/skin.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+
+class Skin(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(name='skin', description='Show a skin of a Minecraft User')
+    async def skincmd(self, ctx, *, username: str = None):
+        if username is None:
+            # Defines the embed and sets basic options
```
Comment (chachyyyy) [inline cogs/skin.py]: do not document code - geek

File: cogs/skin.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            embed = discord.Embed(colour=discord.Color.red(
+            ), title='Error!', description="You need to supply a username!")
+
+            embed.set_footer(text=f"Requested by: {ctx.author.name}#{ctx.author.discriminator}", icon_url=str(
+                ctx.author.avatar_url))  # Sets the footer of the embed
```
Comment (chachyyyy) [inline cogs/skin.py]: dont make a new class - geek

**Comment by GamingGeek** [comment]
The code doesn't really fit my code style and I don't think it should have it's own file.

File: cogs/skin.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+class Skin(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(name='skin', description='Show a skin of a Minecraft User')
+    async def skincmd(self, ctx, *, username: str = None):
+        if username is None:
+            # Defines the embed and sets basic options
+            embed = discord.Embed(colour=discord.Color.red(
```
Comment (chachyyyy) [inline cogs/skin.py]: something about dont use discord.Color - geek

**Comment by GamingGeek** [comment]
you also commited .vscode/settings.json

**Pull request closed without merging**